particula/util/wall_loss.py

```diff
@@ -9,41 +9,147 @@
 from particula.util.diffusion_coefficient import pdc
 
 
+def spherical_wall_loss_coefficient(
+    ktp_value,
+    diffusion_coefficient_value,
+    chamber_radius,
+    settling_velocity_value
+):
+    """Calculate the wall loss coefficient for a spherical chamber
+        approximation.
+
+    Args:
+        ktp_value: ?? turbulent parameter ??
```

Comment:
@ngmahfouz is this the same kt as you wrote in the issue, and what does it mean? or is it just some aerosol specific parameter.